internal/pkg/deploy/cloudformation/stack/testdata/workloads/svc-prod.stack.yml

```diff
@@ -396,8 +396,8 @@ Resources: # If a bucket URL is specified, that means the template exists.
       TargetTrackingScalingPolicyConfiguration:
         PredefinedMetricSpecification:
           PredefinedMetricType: ECSServiceAverageCPUUtilization
-        ScaleInCooldown: 120
-        ScaleOutCooldown: 60
+        ScaleInCooldown: 30
+        ScaleOutCooldown: 150
```

Comment:
I can certainly change this behavior, but I intended it to be so that the environment will replace an levels higher than it, but not lower than it. So if the environment specified something like:
`count:
  cpu_percentage:
    cooldown:
      out: 1m30s`
then it would be 90. Now, I don't know if this is the behavior that we want to give customers, but it's what made the most sense to me when writing it.

Reply:
no, i think that makes sense to me - i didn't notice the different levels.🙄 this is a great test!😊